### PR TITLE
Translate some hard-coded strings in UI

### PR DESCRIPTION
**Pull request by numero-six**
Translations not included, will do via POEditor as requested.

### CODE REVIEW

File: public/views/codimd/body.ejs [hunk 1]
```diff
@@ -15,27 +15,27 @@
...
                         <li class="divider"></li>
-                        <li class="ui-delete-note"><a><i class="fa fa-trash-o fa-fw"></i> Delete this note</a></li>
+                        <li class="ui-delete-note"><a><i class="fa fa-trash-o fa-fw"></i> <%= __('Delete this note') %></a></li>
                     </ul>
                 </span>
                 <br>
                 <span class="ui-owner" style="display: none;">
                     &thinsp;<i class="ui-user-icon small" data-toggle="tooltip" data-placement="right"></i>
-                    &nbsp;<span class="text-uppercase">owned this note</span>
+                    &nbsp;<span class="text-uppercase"><%= __('owned this note') %></span>
```
Comment (ccoenen) [inline public/views/codimd/body.ejs]: I think this is a typo. It should be "owns this note" instead of "owned".

**Comment by SISheogorath** [comment]
Could you start CodiMD once in development config and open the pages that use the strings? This should update your `./locales/en.json` (which we can import to POEditor for further translations). (Yes, the `en.json` is the only translation file we maintain in the repo to a certain degree)

File: public/views/codimd/body.ejs (hunk 1 above)
Comment (numero-six) [reply public/views/codimd/body.ejs]: I don't know where this message is shown so I'm not sure. Fell free to fix.

**Comment by numero-six** [comment]
> Could you start CodiMD once in development config and open the pages that use the strings? This should update your `./locales/en.json`

Tried that. `en.json` file not updated :disappointed:.

**Comment by SISheogorath** [comment]
```patch
From 23996c902869197b223a27994f696ed49f0a7cf1 Mon Sep 17 00:00:00 2001
From: Sheogorath <sheogorath@shivering-isles.com>
Date: Mon, 24 Feb 2020 13:32:43 +0100
Subject: [PATCH] Add en.json string

Signed-off-by: Sheogorath <sheogorath@shivering-isles.com>
---
 locales/en.json | 11 ++++++++++-
 1 file changed, 10 insertions(+), 1 deletion(-)

diff --git a/locales/en.json b/locales/en.json
index 66f0bced..2b652c1b 100644
--- a/locales/en.json
+++ b/locales/en.json
@@ -118,5 +118,14 @@
     "Powered by %s": "Powered by %s",
     "Help us translating": "Help us translating",
     "Join the community": "Join the community",
-    "Imprint": "Imprint"
+    "Imprint": "Imprint",
+    "Freely - Anyone can edit": "Freely - Anyone can edit",
+    "Editable - Signed-in people can edit": "Editable - Signed-in people can edit",
+    "Limited - Signed-in people can edit (forbid guests)": "Limited - Signed-in people can edit (forbid guests)",
+    "Locked - Only owner can edit": "Locked - Only owner can edit",
+    "Protected - Only owner can edit (forbid guests)": "Protected - Only owner can edit (forbid guests)",
+    "Private - Only owner can view and edit": "Private - Only owner can view and edit",
+    "Delete this note": "Delete this note",
+    "owned this note": "owned this note",
+    "Table of content": "Table of content"
 }
\ No newline at end of file
-- 
2.24.1

```

Hope that helps :)

**Comment by davidmehren** [comment]
Hi @numero-six!
Do you still want to finish this?

**Comment by SISheogorath** [comment]
@davidmehren They are actually translatable now: https://github.com/codimd/server/commit/54bde6b11fa57e2a613cf9b039d5fef65b217e4d

**Comment by davidmehren** [comment]
This PR still adds "Delete this note" and "owned this note". Lets keep this open (at least until 1.6.1 is out the door) and see if @numero-six wants to update this.

**Comment by numero-six** [comment]
> This PR still adds "Delete this note" and "owned this note". Lets keep this open (at least until 1.6.1 is out the door) and see if @numero-six wants to update this.

I will not update it. I've done my part. As I said in [above comment ](https://github.com/codimd/server/pull/277#discussion_r382705734), I don't know the issue raised by @ccoenen is legit so l let you investigate and fix it or ignore it. I'm not working on this PR anymore. I may try to add the french translations later through... Thanks.

**Pull request closed without merging**